source/access/mfa_login.rst

```diff
@@ -3,23 +3,167 @@
+  Avoid using 'CTRL-C', or it will send a ``SIGINT`` signal interrupting
+  your process instead of performing a copy operation.
+
+- From the drop-down menu, choose the institute you are affiliated with.
+  Below, we show an example of a KU Leuven user, but one has to pick the
+  institute he/she is affiliated with.
+
+  .. figure:: mfa_login/vsc_firewall_institute.PNG
+     :alt: Choose your institute
+
+- You will be forwarded to the Identity Provider (IdP) of your institute,
+  and you need to login in a usual way using your registered credentials.
+  For KU Leuven users, the page looks like the following:
+
+  .. _idp_page:
+  .. figure:: mfa_login/idp_page.PNG
+     :alt: idp_page
+
+- If you are already connected to the internal network, then you will be only asked to
+  identify yourself with the MFA authenticator app on your personal phone:
+
+  .. _reauthenticate_phone:
+  .. figure:: mfa_login/reauthenticate_phone.PNG
+     :alt: reauthenticate_phone
+
+  This step may not be necessary when connecting from a white-listed IP address,
+  like the internal networks of the Flemish universities, using a static on-site
+  IP as well as the institutional VPN.
+  For example, if you have already logged upfront into your institution's network
+  then you might not be required to log in again depending on your browser
+  session settings (e.g., accepted cookies).
+
+- Once you are successfully authenticated, you end up on a page telling you that your VSC 
+  identity is confirmed.
+  If you have already performed the previous login in that browser session, you will 
+  immediately end up on this page:
+
+  .. _firewall_confirmed:
+  .. figure:: mfa_login/firewall_confirmed.PNG
+     :alt: firewall_confirmed
+
+- Go back to your browser, SSH client or terminal.
+  You should now be connected to a login node on the cluster or to the OnDemand web portal,
+  respectively.
+  E.g. for a PuTTY session, a successful login is rewarded with
+  a welcome message:
+
+   .. _login_node:
+   .. figure:: mfa_login/login_node.PNG
+      :alt: login_node
+
+That's it! You can continue doing your HPC work as usual.
+
+.. note::
+
+   For ``login.hpc.kuleuven.be`` only, a successful connection will whitelist
+   your IP address for 90 days.
+   Within that time frame, visiting the `VSC firewall page`_ might be optional.
+
+.. _mfa-with-ssh-agent:
+
+Connecting with an SSH agent
+----------------------------
+
+SSH agents can store so-called SSH certificates which various client programs
+(PuTTY, MobaXterm, NoMachine, FileZilla, WinSCP, ...) can then use to
+authenticate.
+Getting an SSH certificate also involves MFA but this only needs
+to performed once since a certificate can be used multiple times as long as it
+remains valid.
+Certain clients (such as :ref:`FileZilla <FileZilla>` or
+:ref:`NoMachine <NX start guide>`) furthermore do not show you the firewall
+link needed for the MFA and hence can only function in combination with an SSH
```

Comment:
can we also add sshfs here as a client ? In the old documentation there was a specific remark for sshfs. I think this is covered if we just list it here that this is a client that also needs a certificate in order to work.

Reply:
Sure. done in ffe64e5